rclcpp_examples/src/parameters/ros2param.cpp

```diff
@@ -36,15 +37,24 @@ parse_args(int argc, char ** argv, std::string & remote_node, param_operation_t
 
   std::string verb = argv[1];
   std::string name = argv[2];
-  size_t slash = name.find('/');
-  if ((slash == std::string::npos) ||
-    (slash == 0) ||
-    (slash == (name.size() - 1)))
-  {
+  std::string variable;
+
+  if (verb == "list") {
+    op = PARAM_LIST;
+    remote_node = name;
     return rclcpp::parameter::ParameterVariant();
+  } else {
```

Comment:
Since the previous line returns I think the `else` is not necessary and the following code should be not nested but one level less indented.